core/arch/aarch64/emit_utils.c

```diff
@@ -119,6 +121,8 @@ insert_exit_stub_other_flags(dcontext_t *dcontext, fragment_t *f,
         /* br x1 */
         *pc++ = 0xd61f0000 | 1 << 5;
     } else {
+        /* We put a NOP here for trace building. */
```

Comment:
Why does trace building need a nop?  I think there's some code missing from this diff (e.g., fixup_indirect_trace_exit): I can't get a clear picture of what the trace building strategy is on A64 vs x86 and why a nop would help.